e2e/cypress/integration/channel/react_to_last_message_shortcut_spec.js

```diff
@@ -0,0 +1,368 @@
+        cy.postMessage(MESSAGES.TINY);
+
+        // # Mouseover the last post and click post comment icon.
+        cy.clickPostCommentIcon();
+
+        // * Check that the RHS is open
+        cy.get('#rhsContainer').should('be.visible');
+
+        // # Post a replies in RHS.
+        cy.postMessageReplyInRHS(MESSAGES.SMALL);
+        cy.postMessageReplyInRHS(MESSAGES.TINY);
+
+        // # Emulate react to last message shortcut when focus is on the right textbox
+        cy.get('#reply_textbox', {timeout: TIMEOUTS.LARGE}).focus().type('{ctrl}{shift}\\');
+
+        // * Check emoji picker opened
+        cy.get('#emojiPicker').should('exist').within(() => {
+        // # Search for an emoji and add it to message
+            cy.findByPlaceholderText('Search Emoji').type('smile{enter}');
+        });
+
+        cy.getLastPostId().then((lastPostId) => {
+            // # Since this was the last post was on the RHS, its the same message center too
+            cy.get(`#${lastPostId}_message`).within(() => {
+                // * Check if reaction was entered on the last message
+                cy.findByLabelText('reactions').should('exist');
+                cy.findByLabelText('remove reaction smile').should('exist');
+            });
+        });
+
+        cy.closeRHS();
+    });
+
+    it('Should open emoji picker for last post on center when RHS is open but focus is not on RHS textbox', () => {
+        // # Making sure there is at least a message to add comment to at center
+        cy.postMessage(MESSAGES.TINY);
+
+        // # Mouseover the entered post and click post comment icon.
+        cy.clickPostCommentIcon();
+
+        // * Check that the RHS is open
+        cy.get('#rhsContainer').should('be.visible');
+
+        // # Post a repliews in RHS.
+        cy.postMessageReplyInRHS(MESSAGES.SMALL);
+        cy.postMessageReplyInRHS(MESSAGES.TINY);
+
+        // # This post is in RHS, where reaction should not be added
+        cy.getLastPostId().then((lastPostId) => {
+            cy.get(`#${lastPostId}_message`).as('postInRHS');
+        });
+
+        // # Incoming posts from other user
+        cy.postMessageAs({sender: users['user-2'], message: MESSAGES.MEDIUM, channelId});
+
+        cy.wait(TIMEOUTS.SMALL);
+
+        // # Click anywhere to take focus away from RHS textbox
+        cy.get('#lhsList').within(() => {
+            cy.findByText('Town Square').click();
+        });
+
+        // # Emulate react to last message shortcut when focus is not on RHS or Center textbox
+        cy.get('body').type('{ctrl}{shift}\\');
+
+        // * Check that emoji picker is opened when focus is not on RHS or CENTER textbox
+        cy.get('#emojiPicker').should('exist');
+
+        // # Close the picker
+        cy.get('body').type('{esc}');
+        cy.wait(TIMEOUTS.TINY);
+
+        // # Focus back on Center textbox and enter shortcut
+        cy.get('#post_textbox', {timeout: TIMEOUTS.LARGE}).focus().type('{ctrl}{shift}\\');
```

Comment:
This step failed on me.  Could you please check.
<img width="512" alt="Screen Shot 2020-01-07 at 5 50 20 PM" src="https://user-images.githubusercontent.com/5334504/71885897-41ea5680-3176-11ea-8450-1a687e3c1728.png">